rules.md

```diff
@@ -66,3 +66,147 @@ and fail unexpectedly when supplied extra long strings.
+##
+
+
+As well as the additional rules coming from Zally, there are also additional Boat Quay rules, some of which are Backbase 
+specific but could be transferable. These additional rules, are documented below.    
+##
+
+## B012: Use Well Understood HTTP Status Codes
+
+Adapted from Zally rule 150. Well-understood codes should be used to ensure that the codes in the spec are being 
+correctly implemented. Any well-understood codes used should be used properly.
+
+### Well-understood codes:
+#### Success Codes
+Code | Method
+-----|-------
+200 | ALL
+201 | POST, PUT
+202 | POST, PUT, DELETE, PATCH
+204 | PUT, DELETE, PATCH
+207 | POST
+
+#### Redirection Codes
+Code | Method
+-----|-------
+301 | ALL
+303 | PATCH, POST, PUT, DELETE
+304 | GET, HEAD
+
+#### Client Side Error Codes
+Code | Method
+-----|-------
+400 | ALL
+401 | ALL
+403 | ALL
+404 | ALL
+405 | ALL
+406 | ALL
+408 | ALL
+409 | POST, PUT, DELETE, PATCH
+410 | ALL
+412 | PUT, DELETE, PATCH
+415 | POST, PUT, DELETE, PATCH
+422 | ALL
+423 | PUT, DELETE, PATCH
+428 | ALL
+429 | ALL
+
+#### Server Side Error Codes
+Code | Method
+-----|-------
+    500 | ALL
+    501 | ALL
+    502 | ALL
+    503 | ALL
+    504 | ALL
+
+## B011: Use standard HTTP status codes
+
+Adapted from Zally rule 150, this adds extra HTTP codes to the list of standard and well-understood codes. Any HTTP codes 
+used in responses must be from the standard list.
+Standard HTTP status codes:
+
+    100, 101, 200, 201, 202, 203, 204, 205, 206, 207
+    300, 301, 302, 303, 304, 305, 307, 400, 401, 402, 
+    403, 404, 405, 406, 407, 408, 409, 410, 411, 412, 
+    413, 414, 415, 416, 417, 422, 423, 426, 428, 429, 
+    431, 500, 501, 502, 503, 504, 505, 511, default
+    
+## B010: Pluralize resource names
+
+Resource names should be pluralised in paths.
+Path components after prefix and version should be plurals, for example, `/client-api/v1/bars` client and v1 are ignored.
+
+## B009: Check prefix for paths should contain version
+
+URL Paths Must contain a version number to maintain a consistent method of versioning though the apis.
+## B008: Check x-icon value in the info block
+
+An x-icon should be provided in the info block of the spec with the assigned value for the API. This should be provided 
+so that the api portal can display the api with the correct icon.
+
+## B007: Check prefix for paths
+
+Path prefixes should be from this set of valid prefixes as it should refer to the type of api.
+
+Valid prefixes:
+ - client-api
+ - service-api
+ - integration-api
+ 
+## B006: Check info block description format
+
+Descriptions are important to include for readability and understanding the purpose of a spec.
+A description must be present in the info block, it cannot be empty and must be no longer then the configured max length 
+which is currently set to 140. 
+
+## B005: Check info block title format
+
+For Readability the title Must not be empty and should be no longer then the configured max title length, 35.
```

Comment:
Readability -> readability  and Must -> must
In the block above they are starting from lowercase :)